common/network-defaults/src/eth_contract.rs

```diff
@@ -0,0 +1,132 @@
+// Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+// This should be modified whenever an updated Ethereum contract is uploaded
+pub const ETH_JSON_ABI: &str = r#"
```

Comment:
The reason I prefer it to being hardcoded in Rust code is that it can easily be compiled in on any platform, without having to mess around with OS file systems.